src/components/Footer/Footer.tsx

```diff
@@ -1,41 +1,46 @@
-import { Group, Anchor } from "@mantine/core";
+import { Container, Group, Anchor } from "@mantine/core";
 import "@mantine/core/styles.layer.css";
 import "./Footer.css";
 import Image from "next/image";
 
+const links = [
+  { link: "/about", label: "About" },
+  { link: "/projects", label: "Projects" },
+  { link: "/team", label: "Team" },
+  { link: "/chapters", label: "Chapters" },
+  { link: "/join", label: "Join" },
+];
+
 const Footer = () => {
+  const items = links.map((link) => (
+    <Anchor<"a">
+      c="white"
+      key={link.label}
+      href={link.link}
+      onClick={(event) => event.preventDefault()}
+      size="sm"
+      className="footer-link"
+    >
+      {link.label}
+    </Anchor>
+  ));
+
   return (
-    <footer className="main-footer">
-      <Group className="footer-menu">
+    <div className="main-footer">
```

Comment:
Lets keep this div called footer. Its an accessibility thing and good to have.

It works the same as div